src/sql/src/session/vars.rs

```diff
@@ -1483,6 +1483,14 @@ pub const ENABLE_COLUMNATION_LGALLOC: ServerVar<bool> = ServerVar {
     internal: true,
 };
 
+pub const ENABLE_EAGER_DELTA_JOINS: ServerVar<bool> = ServerVar {
```

Comment:
This can be a regular `feature_flag!` entry at the moment. The only thing to note is that `enable_for_item_parsing` should be set to `false` as a precaution for this entry.

Reply:
Okay, done!